### PR TITLE
[110] The Software Developer Conditionals

### PR DESCRIPTION
**Pull request by ToluEnny24**
I have created a program that prints numbers between 1 and 116 following conditions

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/javascript/conditionals.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+
+for (var x = 1; x <= 116; x++)
```
[the remark this answers is not in the document]
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/javascript/conditionals.js]: @adaugochi i have performed the difference and used the absolute value

File: submissions/fabunmiTolulope/javascript/conditionals.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+let w = 7;
+let y = 123;
+z = y-w;
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/javascript/conditionals.js:4]: It is advisable to always use well descriptive variable names. things like `y, w, x, a, b` and so on are not conventional ways of naming things. imagine entering a codebase and you start seeing names like that. You will find it difficult reading that code. For me, I would have said `let num1 = 7; let num2 = 123;` then `let diff = num2 - num1`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/javascript/conditionals.js (hunk 2 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/javascript/conditionals.js]: Thanks for the observation, I have used more descriptive words for my variables to make it more readable.

File: submissions/fabunmiTolulope/javascript/conditionals.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+let number1 = 7;
+let number2 = 123;
+positiveDiff = number2 - number1;
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/javascript/conditionals.js:4]: use let or var here in declaring this variable

File: submissions/fabunmiTolulope/javascript/conditionals.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+// prog is the programmed variable
+
+for (var prog = 1; prog <= positiveDiff; prog++) 
+{
+	if (prog% 3 === 0 && prog% 5 === 0)
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/javascript/conditionals.js:9]: the `prog` and `%` are joined together. kindly separate it

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/javascript/conditionals.js (hunk 3 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/javascript/conditionals.js]: @adaugochi I have declared the variable using let.

File: submissions/fabunmiTolulope/javascript/conditionals.js (hunk 4 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/javascript/conditionals.js]: @adaugochi I have seperated them.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**